chromium_src/chrome/browser/extensions/chrome_content_verifier_delegate.cc

```diff
@@ -37,7 +37,7 @@
 #include "extensions/common/manifest_url_handlers.h"
 #include "net/base/backoff_entry.h"
 
-#if BUILDFLAG(IS_CHROMEOS_ASH)
+#if BUILDFLAG(IS_CHROMEOS)
```

Comment:
we really don't need these chromeos flags

Reply:
Removed in add6e384eacae2e7dcc2622575f3815914158a3e.